Interlace/lib/core/input.py

```diff
@@ -207,6 +225,20 @@ def process_commands(arguments):
             else:
                 protocols = arguments.proto
             final_commands = InputHelper._replace_variable_for_commands(final_commands, "_proto_", protocols)
+        
+        # process proxies
+        if arguments.proxy_list:
+            proxy_list = list()
```

Comment:
What do you think about passing all of this through a set to strip out duplicates? I'm unsure, I wonder if some people will intentionally want multiple copies of one proxy (to favor it over others)?

Reply:
This was done intentionally so that in case you have 10 proxies and 100 targets, we can reuse the 10. Also, this allows a tester to weight their proxies as per the load required. Say I have 5 proxies, and I want 1 proxy to take a majority of the load, so I'll add the proxy an extra 3 times to the file.